pagopa/io-app
# feat: [IA-836] Integrate the new "Spunta Blu" service preference

## Short description
This PR is going to integrate the new toggle in the services preferences for the "Spunta Blu" premium feature.

⚠️ Depends on https://github.com/pagopa/io-dev-api-server/pull/179

https://user-images.githubusercontent.com/5963683/173793386-ec385450-1cc1-4077-b25b-b3b64c4004ec.mov

## List of changes proposed in this pull request
- Handled the new `can_access_message_read_status` service preference
- Created a new toggle with the same behaviour as the one used for the push notifications

## How to test
First of all, enable the `isPremiumMessagesOptInOutEnabledSelector` flag, then using the [new version of the `io-dev-api-server`](https://github.com/pagopa/io-dev-api-server/pull/179) it should be possible to see the new toggle in the single service page (just among the other preferences). This new toggle should have the _same exact behaviour_ as the one used for the push notifications.
